utils/polus-bfio-util/bfio/bfio.py

```diff
@@ -18,32 +18,59 @@
 import os
 import javabridge as jutil
 
+
 def make_ome_tiff_writer_class():
     '''Return a class that wraps loci.formats.out.OMETiffWriter'''
     class_name = 'loci/formats/out/OMETiffWriter'
-    IFormatWriter = bioformats.formatwriter.make_iformat_writer_class(class_name)
+    IFormatWriter = bioformats.formatwriter.make_iformat_writer_class(
+        class_name)
 
     class OMETiffWriter(IFormatWriter):
 
         new_fn = jutil.make_new('loci/formats/out/OMETiffWriter', '()V')
-        
+
         def __init__(self):
 
             self.new_fn()
 
     return OMETiffWriter
 
+
 class BioReader():
-    # Note: the javabridge connection must be started before initializing a 
-    # BioReader object. The reason for this is that the java VM can only be
-    # initialized once. Doing it within the code would cause it to run possibly
-    # without shutting down, so the javabridge connection must be handled
-    # outside of the class.
+    """BioReader Read supported image formats using Bioformats
+
+    This class handles reading data from any of the formats supported by the 
+    OME Bioformats tool. It handles some of the bugs that are commonly
+    encountered when handling larger images, such as the indexing issue
+    encountered when an image plan is larger than 2GB.
+    
+    Note: The javabridge is not handled by the BioReader class. It must be
+          initialized prior to using the BioReader class, and must be closed
+          before the program terminates. An example is provided in read_image().
```

Comment:
I think we should make this point (especially about closing) as not a note but something more prominent. Its important for younger devs to know they must close it.

Reply:
I changed it to WARNING